rules/show-certification-award/rule.md

```diff
@@ -0,0 +1,39 @@
+---
+type: rule
+title: Do you show Certification Reward?
+uri: show-certification-award
+authors:
+  - title: Stephan Fako
+    url: https://www.ssw.com.au/people/stephan-fako/
+related: []
+created: 2024-04-08T06:55:48.587Z
+guid: 94ab00f4-a6d5-4e8f-aca3-68d6cc0a3498
+---
+**Do you show Certification Reward?**
+ 
+Encouraging professional development and recognizing achievements are crucial for fostering a positive workplace culture. SSW's Certification Reward system is a testament to this, offering special leave to employees who complete Microsoft certifications. This initiative not only motivates employees to pursue further learning but also highlights the company's commitment to staying at the forefront of technology.
+ 
+---
+ 
+**Understanding Certification Reward**
+ 
+Certification Reward is a special category of leave granted to SSW employees as a form of recognition for their effort and time invested in completing Microsoft certifications. Here's how it works:
+ 
+1. Award Criteria: Employees become eligible for Certification Reward upon successfully completing a Microsoft certification.
+2. Visibility and Tracking: SSW utilizes a distinct leave category within its accounting system specifically for Certification Reward. 
+ 
+---
+ 
+**Not a Fairwork Mandate**
+ 
+It's important to note that the Certification Reward is not mandated by Fairwork. Instead, it's a proactive approach by SSW to incentivize its team members to engage in professional development, specifically around Microsoft technologies. This strategic focus not only enhances individual skill sets but also elevates the collective expertise within the company, ensuring SSW remains competitive and innovative in delivering solutions.
+ 
+---
+ 
+**Benefits of Certification Reward**
+ 
+✅ Professional Development: Encourages employees to acquire new skills and knowledge, staying current with the latest technology trends  
```

Comment:
Hi @stephanfakossw 

- [ ] 1. In this list, can you make the words before ":" in bold? It will help readability
- [ ] 2. At the end of each of this list items... please add a double space... this will make the line break after them

Reply:
> 2. The lines already have a double space :-)